src/main/java/com/aws/greengrass/easysetup/DeviceProvisioningHelper.java

```diff
@@ -366,7 +369,7 @@ public void updateKernelConfigWithIotConfiguration(Kernel kernel, ThingInfo thin
         }
 
         Path caFilePath = certPath.resolve("rootCA.pem");
-        downloadRootCAToFile(caFilePath.toFile());
+        downloadCAToFile(caFilePath.toFile(), ROOT_CA_URL, CA_3_URL);
```

Comment:
Why not initialize a string array as a class variable and use it in `downloadCAToFile` instead of passing in multiple parameters?

`String[] caUrls = new String[] {ROOT_CA_URL, CA_3_URL};`

Reply:
The function `downloadCAToFile(File f, String... urls)` is accepting the url parameter as a String array